packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -68,12 +69,18 @@ export class ZipkinExporter implements SpanExporter {
     spans: ReadableSpan[],
     resultCallback: (result: ExportResult) => void
   ) {
+    if (typeof this._serviceName !== 'string') {
+      this._serviceName = String(
+        spans[0].resource.labels[SERVICE_RESOURCE.NAME] ||
+          'OpenTelemetry Service'
```

Comment:
I would prefer to have `const` for this.

Reply:
do you mean like ```const defaultServiceName = String(
        spans[0].resource.labels[SERVICE_RESOURCE.NAME] ||
          'OpenTelemetry Service');```?

Reply:
No he means defining a constant somewhere like `const DEFAULT_SERVICE_NAME = 'OpenTelemetry Service'` then doing

```
String(spans[0].resource.labels[SERVICE_RESOURCE.NAME] || DEFAULT_SERVICE_NAME);
```